src/Sylius/Bundle/ApiBundle/SerializerContextBuilder/LocaleContextBuilder.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\SerializerContextBuilder;
+
+use ApiPlatform\Core\Serializer\SerializerContextBuilderInterface;
+use Sylius\Bundle\ApiBundle\Serializer\ContextKeys;
+use Sylius\Component\Locale\Context\LocaleContextInterface;
+use Sylius\Component\Locale\Context\LocaleNotFoundException;
+use Sylius\Component\Resource\Translation\Provider\TranslationLocaleProviderInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+final class LocaleContextBuilder implements SerializerContextBuilderInterface
+{
+    private $decorated;
+    private $localeContext;
+    private $translationLocaleProvider;
+
+    public function __construct(SerializerContextBuilderInterface $decorated, LocaleContextInterface $localeContext, TranslationLocaleProviderInterface $translationLocaleProvider)
+    {
+        $this->decorated = $decorated;
```

Comment:
Change `decorated` to `decoratedContextBuilder` the same as I've commented you in `ChannelContextBuilder`